### PR TITLE
[ADD] crm_deduplicate_filter

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Exclude some records from the deduplication

This module extends the possibilities of the contact deduplication allowing
to filter the applicable set according to several criteria.

For now, only the filter for restricting the deduplication to only companies or
only contacts is implemented.
# Usage

To use this module, you need to:

   "Parent company set (Contacts)" in the section 'Exclude contacts having'.
   kind of records.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: crm_deduplicate_filter/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Exclude some records from the deduplication
+===========================================
+
+This module extends the possibilities of the contact deduplication allowing
+to filter the applicable set according several criteria.
```
Comment (elicoidal) [inline crm_deduplicate_filter/README.rst]: according to

**Comment by elicoidal** [comment]
:+1:

File: crm_deduplicate_filter/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+===========================================
+Exclude some records from the deduplication
+===========================================
+
+This module extends the possibilities of the contact deduplication allowing
+to filter the applicable set according to several criteria.
+
+For now, only the filter for restricting the deduplication to only companies or
+only contacts is implemented.
```
Comment (yajo) [inline crm_deduplicate_filter/README.rst:13]: This should go to known issues instead

**Comment by yajo** [comment]
:+1:

**Comment by rafaelbn** [comment]
@pedrobaeza if you review comments we can merge this one

File: crm_deduplicate_filter/README.rst (hunk 2 above)
Comment (pedrobaeza) [reply crm_deduplicate_filter/README.rst:13]: That's not a known issue, but only the scope. The known issue / roadmap should be "Implement more filters for deduplication", but I decided to let it this way.

**Pull request merged**